### ISSUE TITLE
Swinging_Chassis handling flag does not work

### ISSUE THREAD

**Issue by KillerHDTR** [multitheftauto/mtasa-blue#1117]
**Describe the bug**
I tried to disable body shaking for cars but Swinging_chassis handling flag does not apply any effect, I mean if i disable swinging_chassis flag the car body is shaking again.

**To reproduce**
Get a vehicle that has swinging chassis (e.g Greenwood), open hedit and untick the swinging_chassis flag, drive around and make some turns with your car and you can see chassis is still shaking

**Expected behaviour**
If i disable swinging_chassis using the handling flag, the car body should not be able to shake

**Staff note** (15th Feb 2020)
Cleaned up the issue description.

MTA doesn't listen to the value of swinging chassis flag. This is a bug in MTA, it works in GTA. It would probably be a good first issue for someone.

**Version**
All MTA versions, up until as of writing (MTA 1.5.7)

**Comment by qaisjp** [comment]
Related to https://bugs.mtasa.com/view.php?id=6529

**Comment by KillerHDTR** [comment]
So, what does that mean?

**Comment by Brian9221** [comment]
The only way to make a car stop "shaking" is to rename internal model name from "chassis" to something different. This requires some 3D modeling software access and to apply a mod over the model you want to stop "shaking".

**Comment by tut001** [comment]
> The only way to make a car stop "shaking" is to rename internal model name from "chassis" to something different. This requires some 3D modeling software access and to apply a mod over the model you want to stop "shaking".

Thanks for the tip! I'll definitely keep it in mind as temporary solution if it doesn't cause conflicts with our server's resources by changing the name.

**Comment by AlexTMjugador** [comment]
> The only way to make a car stop "shaking" is to rename internal model name from "chassis" to something different. This requires some 3D modeling software access and to apply a mod over the model you want to stop "shaking".

Out of curiosity, does that rename have any side effect? For instance, I know that the chassis_vlo is used as a low LOD mesh, so how does that renaming interact with the GTA LOD system?

**Comment by Brian9221** [comment]
Nope, there aren't any visible issues to the game. You'll keep seeing the vehicle's LOD and every other feature related to it.

**Comment by Dutchman101** [comment]
> Nope, there aren't any visible issues to the game. You'll keep seeing the vehicle's LOD and every other feature related to it.

Still though, work arounds by changing stuff in the model is basically confusing the SA engine, and comes down to deliberately corrupting models in such a way that swinging chassis (a GTA feature) won't work no matter what

Manipulation of models isn't recommended practise, it can lead to unknown side effects. Better wait until MTA fixes this bug, so that the game reacts to modification of the Swinging chassis flag like it should.

Note: i heard someone else about this method, 

>  turned out all i have to do is put wheels to parent instead of inside chassis_dummy

yet again, not recommended.
